src/shogun/lib/type_case.h

```diff
@@ -38,8 +42,10 @@ namespace shogun
 		PT_FLOATMAX = 13,
 		PT_SGOBJECT = 14,
 		PT_COMPLEX128 = 15,
-		PT_UNDEFINED = 16
+		PT_SGVECTOR_FLOAT32 = 16,
```

Comment:
PT stands for "primitive type" in the old parameter framework. So keeping that in here doesnt make sense, maybe we need a new prefix?

Reply:
Could just leave without a prefix? Otherwise just use SG_ instead?

Reply:
what about T_

Reply:
Yup, can use that